doc/usermanual/lighttable/panels/tagging.xml

```diff
@@ -193,8 +193,8 @@
       </para>
 
       <para>
-        You can resize the height of attached view or dictionary view using Ctrl+wheel.
-        New sizes are kept for ulterior darktable sessions.
+        You can dynamically resize the height of attached tags and tag dictionary views using Ctrl+wheel. 
```

Comment:
I'd tend to say "the" attached tag and tag dictionary views. To be confirmed.

Reply:
I rephrased. Please check if it looks better.